scp/js/dashboard.inc.js

```diff
@@ -194,30 +198,31 @@
                         $('<td>').attr('colspan','8').append(
                             'No data for this timeframe found'))).appendTo(q);
                 }
+                $('tbody[page=1]', q).show();
                 $('#table-here').empty().append(q);
 
                 // ----------------------> Pagination <---------------------
                 function goabs(e) {
-                    $('tbody', q).addClass('hide');
+                    $('tbody', q).hide();
                     if (e.target) {
                         page = e.target.text;
-                        $('tbody[page='+page+']', q).removeClass('hide');
+                        $('tbody[page='+page+']', q).show();
                     } else {
-                        e.removeClass('hide');
+                        e.show();
                         page = e.attr('page')
                     }
-                    enable_next_prev(page);
+                    return enable_next_prev(page);
                 }
                 function goprev() {
                     current = $('tbody:not(.hide)', q).attr('page');
                     page = Math.max(1, parseInt(current) - 1);
-                    goabs($('tbody[page='+page+']', q));
+                    return goabs($('tbody[page='+page+']', q));
                 }
                 function gonext() {
                     current = $('tbody:not(.hide)', q).attr('page');
                     page = Math.min(Math.floor(json.data.length / pagesize) + 1,
                         parseInt(current) + 1);
-                    goabs($('tbody[page='+page+']', q));
+                    return goabs($('tbody[page='+page+']', q));
```

Comment:
`gonext` is used in a javascript event routine, which returning `false` indicates that the default event handler should not be invoked.